.docker/php/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN mkdir -p /assets && mv /work/build/* /assets
 
 
 # Build the PHP container
-FROM php:8.0-fpm-alpine
+FROM php:8.1-fpm-alpine
```

Comment:
I don't think we have a CI step for verifying this :D 

In fact, no CI at all?